asaptools/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
FYI, unless you have a strong reason to keep this here, I'd recommend just keeping it in the root as you also have it. There, it will automatically be included in source distributions and binary wheels when you package this in PyPI and conda (thanks in part, incidentally, to the small contributions I made to both pip and conda-build), and is where Github, build/packaging systems and other tools expect it. Keeping it here also is just duplication and a potential source of trouble, and it won't actually be included in either source or binary distributions (PyPI, wheel, conda, etc.) unless you explicitly set up MANIFEST.in and setup.py to include it (whereas they automatically include it from the root with no extra effort).

Reply:
Yeah.  That's a legacy from before PyPI's automatic inclusion.  It would be nice to remove it.